### PR TITLE
Add OpenFHE to NF GSoC 2023

### PR DESCRIPTION
**Pull request by iquah1**

### CODE REVIEW

**Comment by iquah1** [comment]
@MridulS 

here you go!

**Review by henrykironde — COMMENTED** [review]
Check the order of the list and place it in the right place.

File: 2023/ideas-list.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ page of each organization under the NumFOCUS umbrella at this page.
 - [JuMP](https://github.com/jump-dev/GSOC2023)
 - [matplotlib](https://github.com/matplotlib/matplotlib/wiki/GSoC-2023-Ideas)
 - [Mesa](https://github.com/projectmesa/mesa/wiki/Projects-for-Google-Summer-of-Code-2023)
+- [OpenFHE](https://github.com/openfheorg/openfhe-development/wiki/GSOC-2023)
```
[the remark this answers is not in the document]
Comment (iquah1) [reply 2023/ideas-list.md]: Done! Thank you for catching that

**Comment by MridulS** [comment]
Thanks!

**Pull request merged**